src/languageservice/parser/jsonParser07.ts

```diff
@@ -767,6 +767,7 @@ function validate(
         matchingSchemas: ISchemaCollector;
       } = null;
 
+      let alternativesFiltered = alternatives;
```

Comment:
only for better debugging, to be able display/debug original and filtered collection